frontend/app/src/pages/settings/api-keys/external/index.vue

```diff
@@ -151,9 +164,11 @@ const confirm = async (service: ExternalServiceName) => {
   set(loading, false);
 };
 
+const getName = (chain: string) => get(getChainName(chain as Blockchain));
```

Comment:
Btw we don't have to do anything for this PR but I think we should have a different approach with this.

@rotki/frontend 

We can use plain functions and [reactify](https://vueuse.org/shared/reactify/) when needed.

This way we can also avoid that thing here.

Also useful is this [createUnrefFn](https://vueuse.org/core/createUnrefFn/) which makes it so that you can pass refs to a normal non-reactive function.

So the approach should be to use plain functions and use either of the utilities above when needed.